sui-execution/next-vm/sui-adapter/src/temporary_store.rs

```diff
@@ -1095,6 +1095,13 @@ impl<'backing> Storage for TemporaryStore<'backing> {
     ) {
         TemporaryStore::save_loaded_runtime_objects(self, loaded_runtime_objects)
     }
+
+    fn save_wrapped_object_containers(
+        &mut self,
+        _wrapped_object_containers: BTreeMap<ObjectID, ObjectID>,
+    ) {
+        panic!("Unused in v1")
```

Comment:
Is this the canonical way of doing this?

Reply:
I always use `unreachable!("<reason>")` since I feel as though it's a bit more clear than a `panic`. Although this should have the implementation from latest and not this...